content/docs/1.3.1/advanced-resources/data-recovery/recover-without-system.md

```diff
@@ -11,13 +11,26 @@ This command gives users the ability to restore a backup to a `raw` image or a `
 
 3. Specify the host path of output file by modifying field `hostpath` of volume `disk-directory`. By default the directory is `/tmp/restore/`.
 
-4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://backupbucket@us-east-1/backupstore?backup=backup-bd326da2c4414b02&volume=volumeexamplename`. Do not delete `''`.
+4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://<your-bucket-name>@<your-aws-region>/backupstore?backup=<backup-name>&volume=<volume-name>`. Do not delete `''`.
+
+    - `<backup-name>` and `<volume-name>` can be retrieved from backup.cfg stored in the backup destination folder, e.g. `backup_backup-72bcbdad913546cf.cfg`. The content will be like below: 
+
+        <pre><code>{"Name":"backup-72bcbdad913546cf","VolumeName":"volume_1","SnapshotName":"79758033-a670-4724-906f-41921f53c475"......}</code></pre>
 
 5. Set argument `output-file` by replacing `<OUTPUT_FILE>`, e.g. `volume.raw` or `volume.qcow2`.
 
 6. Set argument `output-format` by replacing `<OUTPUT_FORMAT>`. The supported options are `raw` or `qcow2`.
 
-7. Set the S3 Credential Secret by replacing `<S3_SECRET_NAME>`, e.g. `minio-secret`. 
+7. Set the S3 Credential Secret by replacing `<S3_SECRET_NAME>`, e.g. `minio-secret`.  
+
+    - The credential secret could reference [here](https://longhorn.io/docs/{{< current-version >}}/snapshots-and-backups/backup-and-restore/set-backup-target/#set-up-aws-s3-backupstore) and created in the `longhorn-system' namespace.
+
+        ```shell
+        kubectl create secret generic <minio-secret> \
+            --from-literal=AWS_ACCESS_KEY_ID=<your-aws-access-key-id> \
+            --from-literal=AWS_SECRET_ACCESS_KEY=<your-aws-secret-access-key> \
+            -n longhorn-system
+        ```
```

Comment:
This has been mentioned in the link you added, so don't think we need to repeat it here again.